Sources/SnapshotTesting/Snapshotting/UIImage.swift

```diff
@@ -2,6 +2,13 @@
   import UIKit
   import XCTest
 
+  public extension Diffing where Value == UIImage {
+    /// A function to calculate diff image
+    static var findDiff: (_ oldImage: UIImage, _ newImage: UIImage) -> UIImage = { oldImage, newImage in
+      SnapshotTesting.dgisPixelDiff(oldImage, newImage) ?? newImage
+    }
```

Comment:
Added as a static property so that we could change the implementation of the diff function from our code